layouts/shortcodes/cve-feed.html

```diff
@@ -21,3 +22,35 @@
   {{ end }}
   </tbody>
 </table>
+
+<script>
+  window.addEventListener("DOMContentLoaded", () => {
+    const table = document.querySelector("table.security-cves")
+    const tbody = table.querySelector("tbody")
+    const cachedInnerHTML = tbody.innerHTML
+  
+    const renderCVEs = (cves) => {
+      let tbodyHTML = ""
+      for (let cve of cves) {
+        tbodyHTML += `<tr>
+          <td><a href="${cve.url}">${cve.id}</a></td>
+          <td>${cve.summary}</td>
+          <td><a href="${cve.url}">#${cve["_kubernetes_io"].issue_number}</a></td>
+        </tr>`
+      } 
+      return tbodyHTML
+    } 
+
+    setInterval(() => {
+      fetch(`${window.location.origin}/docs/reference/issues-security/official-cve-feed/index.json`)
```

Comment:
Here's a problem: the HTML and the JSON data at that URL get updated in lock step. So this change by itself wouldn't cut down the latency of updates.

What I think we actually want to fetch is `.Site.Params.cveFeedBucket`. I'd do that on page load and then once every 10 minutes.

Reply:
Hi @sftim , I implemented your recommendations.